### DISCUSSION TITLE
[Feature] Tag on multi selection

### DISCUSSION THREAD

**Discussion by IMBeniamin** [immich-app/immich#3522] [Feature Request]
### The feature

Add a button to add that allows to add a tag currently selected photos.
![Example of the context in which this feature could be used](https://github.com/immich-app/immich/assets/35833890/206b9e4c-3b3e-43df-b9c1-6f7da37644be)
Example of the context in which this feature could be used

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by daniele-athome** [comment]
Can you elaborate on what a tag is? Do you mean a label of sort?

**Reply by IMBeniamin** [reply]
Yes, I am referring to a label.
I thought that immich already had a tag system, but it doesn't seem to have, or at least it's not accessible through the ui, so for now this feature is impossible to implement.

A tag system in my opinion would be pretty useful for organization and searching.
Object identification and Face Recognition for example could be seen as a tagging system, as they "attach" or correlate object/face data to the photo.

Ideally a photo should allow for multiple tags.

eg. The photos presented in the screenshot I sent could have the following tags: [apartment, y's_house, interior_design_ideas]

**Reply by IMBeniamin** [reply]
Tags have also been mentioned in this discussion #1651

**Reply by Qhilm** [reply]
Tags are now supported on Immich.

[edit] It _is_ possible to apply a tag to a multi selection. What I miss is the ability to edit existing tags. I was trying to apply a tag to a multi-selection while looking at the pictures under a specific tag – that is still not possible, it would have been a workaround for the lack of tag editing feature (as in, change the tag name).
